src/main/java/org/ow2/proactive/catalog/callgraph/CatalogObjectCallGraphPDFGenerator.java

```diff
@@ -263,4 +266,25 @@ private void buildCatalogCallGraph(List<CatalogObjectMetadata> catalogObjectMeta
 
     }
 
+    private class mxGraphComponentWithoutDragAndDrop extends mxGraphComponent {
```

Comment:
Java doc would be very helpful to understand the new added class

Reply:
I agree with Mohamed it would be good to add comments explaining why those two class were added.